### PR TITLE
added remaining functions to s_y

### PR DESCRIPTION
**Pull request by ianaf100**
Entered remaining functions from function_maps.rs like log, ln, csc, etc. Removed function_maps and updated tests

### CODE REVIEW

File: src/s_y.rs [hunk 1]
```diff
@@ -214,11 +214,36 @@ pub fn calculate(input: &Vec<Token>) -> Result<f32, Vec<String>> {
...
                 match &function_name as &str {
-                    "cos" => {
+                    "log" => {
                         let arg = stack.pop();
-
-                        if let Some(Token::DecimalNumber(n1)) = arg {
-                            stack.push(Token::DecimalNumber(n1.cos()));
+                        match arg {
+                            Some(Token::DecimalNumber(n)) => stack.push(Token::DecimalNumber((n as f32).log10())),
+                            Some(Token::WholeNumber(n)) => stack.push(Token::DecimalNumber((n as f32).log10())),
```
Comment (TPXS-Drexel) [inline src/s_y.rs:221]: do we want to make a second log function for things other than base 10?/

File: src/s_y.rs [hunk 2]
```diff
@@ -214,11 +214,36 @@ pub fn calculate(input: &Vec<Token>) -> Result<f32, Vec<String>> {
...
+                    },
+                    "ln" => {
+                        let arg = stack.pop();
+                        match arg {
+                            Some(Token::DecimalNumber(n)) => stack.push(Token::DecimalNumber((n as f32).ln())),
+                            Some(Token::WholeNumber(n)) => stack.push(Token::DecimalNumber((n as f32).ln())),
+                            _ => ()
+                        }
+                    },
+                    "sqrt" => {
```
Comment (TPXS-Drexel) [inline src/s_y.rs:235]: should have some negative check to put 'i' outside the sqrt

**Review by TPXS-Drexel — COMMENTED** [review]
see comments, then merge as needed

File: src/s_y.rs (hunk 1 above)
Comment (ianaf100) [reply src/s_y.rs:221]: Sure, there's a function for it, I just wasn't sure if a TI could do that. Would this function then be called log10?

Comment (TPXS-Drexel) [reply src/s_y.rs:221]: yeah, the nomenclature on a ti is log(n, base) we should probably stick to log 10 for now as it requires a function of multiple arguments which the shunting yard doesnt have yet, add it as a //TODO:

File: src/s_y.rs (hunk 2 above)
Comment (TPXS-Drexel) [reply src/s_y.rs:235]: this can also be a //TODO:

**Pull request merged**